CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+7.1.0
+-----
+- Datadog user-agent has changed to `gostatsd` by default.  Can now be configured.
+- New Datadog option: `user-agent`, configures the user agent supplied to Datadog.  Use `python-requests/2.6.0 CPython/2.7.10`
+  for old behavior.
```

Comment:
I'm not really sold on the idea that this should be configurable. I can't see a use for this other than to masquerade as the datadog agent and then what we're really trying to do is subvert Datadog's view of which clients access it. I think this is something we could add in down the track if there are good reasons for it (eg. in the unlikely event that someone from Datadog asks us to). It's not likely that Datadog would attempt to block access to its customers by using the User-Agent, and they'd have to have a pretty good reason to do so.

Reply:
I prefer to have it and not need it. It's an extra bit now vs another cycle later.